config/config.php

```diff
@@ -223,6 +223,25 @@
             ],
         ],
 
+        /*
+         * Validators configurations. Corresponding validator will be created and
+         * will be injectable into components.
+         */
+        'validators' => [
+            'complexSetupValidator' => [
+                [
+                    'field' => 'sessions',
+                    'rule' => 'length',
+                    'value' => [0, 2]
```

Comment:
Maybe add a comment here too. What is the sessions field and why must it be between 0 and 2?